lib/constants.js

```diff
@@ -1,6 +1,7 @@
 import path from 'path';
 
 const WDA_RUNNER_BUNDLE_ID = 'com.facebook.WebDriverAgentRunner';
+const WDA_RUNNER_BUNDLE_ID_FOR_XCTEST = 'com.facebook.WebDriverAgentRunner.xctrunner';
```

Comment:
```
const WDA_RUNNER_BUNDLE_ID_FOR_XCTEST = `${WDA_RUNNER_BUNDLE_ID}.xctrunner`;
```

Reply:
not sure if we need a separate constant for it though

Reply:
Either is fine, I wanted to explicitly keep the `xctrunner` as xctest bundle's bundle id